presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -283,7 +283,9 @@ private void printQueryInfo(QueryResults results)
             }
 
             assert terminalWidth >= 75;
-            int progressWidth = (min(terminalWidth, 100) - 75) + 17; // progress bar is 17-42 characters wide
+            String formattedWallTime = formatTime(wallTime);
+            // progress bar is 17-42 characters wide when the wall time has 5 characters (hh:mm)
+            int progressWidth = (min(terminalWidth, 100) - 75) + 22 - formattedWallTime.length();
```

Comment:
Comment explains what 17, 42 and 5 mean but the code uses numbers 100, 75, 22. Would it be possible to correlate code and comment?